resources/src/asset.rs

```diff
@@ -1,6 +1,37 @@
-use serde::{Deserialize, Deserializer};
-use serde::de;
-use std::str::FromStr;
+use deserialize;
+
+/// Assets are the units that are traded on the Stellar Network.
+/// An asset consists of an type, code, and issuer.
+/// Any asset can be traded for any other asset.
+/// https://www.stellar.org/developers/horizon/reference/resources/asset.html
+///
+/// A partial is just the type, code, issuer
+#[derive(Serialize, Deserialize, Debug)]
+pub struct PartialAsset {
```

Comment:
I like this move especially since this is used so frequently in all of the offer payloads.  I might suggest calling this AssetIdentifier seeing as these are the three fields required to uniquely identify an asset in Stellar.  What are your thoughts?

Reply:
Good idea, I like `AssetIdentifier`, I was having trouble coming up with a good name.